src/components/ComponentIndexPage/component-index.scss

```diff
@@ -148,3 +148,28 @@
   color: $text-01;
   margin-bottom: $spacing-06;
 }
+
+//----------------------------------------------------------------------------
+// Component Index Filter
+//----------------------------------------------------------------------------
+.component-index-filter-container {
+  margin-top: 12.5rem;
+}
+// Header
+.component-index-filter__header {
+  @include carbon--type-style('heading-01');
+  color: $text-01;
+}
+.bx--label  {
+  border-top: 1px solid lightgray;
+  padding-top: 1rem;
+}
+.bx--fieldset  {
+  padding-bottom: 1rem;
+  margin-top: 1rem;
+}
+
+// Filter option 
+.component-index-filter__option {
+  padding-bottom: 1rem;
```

Comment:
Might be able to get away with some spacing tokens for some of the `rem` values added 👀